src/navigators/RootStack.tsx

```diff
@@ -84,6 +104,8 @@ export function RootStack() {
       ? 'InviteRequired'
       : shouldRenderConsentScreen
       ? 'screens/ConsentScreen'
+      : shouldLaunchOnboardingCourse
+      ? 'screens/OnboardingCourseScreen'
       : 'app';
```

Comment:
Lot of permutations in here.  New/existing user, has/has not given consent, has/has not seen onboarding course.  @aymanenadi did you manually verify each one works as expected?

Reply:
Yes, I think I covered all these cases during my tests. I believe there is room for improvement here.

Reply:
I was talking to Swain about this a little bit and I think it would be awesome if we could make this configurable by a consumer of the sdk. That way they could use developer config to inject their own initial screens if they want and when they are done they can navigate to the app (possibly via a callback we hand them). Using this system we could probably clean up this logic a bit, make it more deterministic, make it easier to test, and allow for customization.

Reply:
https://lifeomic.atlassian.net/browse/PHC-5160